src/hub/Hub.sol

```diff
@@ -169,15 +169,15 @@ contract Hub is Multicall, Auth, Recoverable, IHub, IHubGatewayHandler {
     //----------------------------------------------------------------------------------------------
     /// @inheritdoc IHub
     function notifyPool(PoolId poolId, uint16 centrifugeId) external payable {
-        _protectedAndPaid(poolId);
+        _isManagerAndPaid(poolId);
```

Comment:
I felt `_protectedAndPaid`/`_protected` was confusing wrt the re-entrancy modifier also called `protected`

Reply:
Much more legible!